ginivision/src/main/java/net/gini/android/vision/digitalinvoice/DigitalInvoiceFragmentListener.kt

```diff
@@ -8,4 +11,10 @@ package net.gini.android.vision.digitalinvoice
 interface DigitalInvoiceFragmentListener {
 
     fun onEditLineItem(selectableLineItem: SelectableLineItem)
+
+    fun onPayInvoice(selectedLineItems: List<LineItem>,
+                     selectedLineItemsTotalPrice: String,
+                     deselectedLineItems: List<LineItem>,
+                     reviewedCompoundExtractions: Map<String, GiniVisionCompoundExtraction>,
+                     reviewedExtractions: Map<String, GiniVisionSpecificExtraction>)
```

Comment:
Will the callee benefit from this partitioning? I don't see some of those parameters used anywhere in the example code.

Reply:
No, I did this only to have things returned until we decide what to return exactly. I will adapt and finalize this in [PIA-540](https://ginis.atlassian.net/browse/PIA-540).